### DISCUSSION TITLE
Using Raspberry PI 3B (not plus)

### DISCUSSION THREAD

**Discussion by odflor** [frntc/RAD#24] [General]
Hi!

I'm using the RAD Project with my Raspberry PI 3B (not plus) SOC.

Also I'm using it with my c128 NTSC "Slim".

Normally I need to power my RbPI with its own power source about of 5A capability.

Even so, I usually experience some gliches.

Is important that I need to use a RbPI 3B+(plus) instead of my RbPI3B (not plus)?

Best Regards

**Comment by frntc** [comment]
If you're lucky and your RPi 3B has the same SoC as the 3B+, then it should work (see other post on timings).

The RPi doesn't need 5A, actually it's within specs of the expansion port when using a single core only (as Sidekick64 and RAD do). But still, a stable power supply is required and I recommend to use one.

**Reply by squelch41** [reply]
3b has been working well for me (Board is marked "3b v 1.2)
